pycalrissian/context.py

```diff
@@ -390,12 +390,22 @@ def create_role_binding(self, name: str, role: str):
 
         role_ref = client.V1RoleRef(api_group="", kind="Role", name=role)
 
-        subject = client.models.RbacV1Subject(
-            api_group="",
-            kind="ServiceAccount",
-            name="default",
-            namespace=self.namespace,
-        )
+        # Check the version of the client to determine the correct class to use
+        from packaging.version import Version
```

Comment:
Move the import to the top of the file with the rest of the imports

Reply:
done